stdlib/random.mli

```diff
@@ -42,6 +42,12 @@ val int : int -> int
      and [bound] (exclusive).  [bound] must be greater than 0 and less
      than 2{^30}. *)
 
+val int63 : int -> int
+(** [Random.int63 bound] is identical to {!int} if [bound] is less than 2{^30},
+    however [bound] may be any positive [int].
```

Comment:
```suggestion
(** [Random.int63 bound] is equal to [{!int} bound] if [bound] is less than
    2{^30}, however [bound] may be any positive [int].
```

Reply:
I would be even more explicit:
```
[Random.int63 bound] returns a random integer between 0 (inclusive)
and [bound] (exclusive). [bound] may be any positive integer. 
If [bound] is less than 2{^30}, [Random.int63 bound] is equal to [{!Random.int} bound].
On a 64-bit platform, [bound] can be greater than 2{^30}, in which case [{!Random.int} bound] would fail but [Random.int63 bound] works. *)
```
The English needs work, but you get the idea: first say what the function does, only then compare and contrast with `Random.int`.

Reply:
How about this?

```diff
--- a/stdlib/random.mli
+++ b/stdlib/random.mli
@@ -43,8 +43,13 @@ val int : int -> int
      than 2{^30}. *)

 val int63 : int -> int
-(** [Random.int63 bound] is identical to {!int} if [bound] is less than 2{^30},
-    however [bound] may be any positive [int].
+(** [Random.int63 bound] returns a random integer between 0 (inclusive)
+     and [bound] (exclusive). [bound] may be any positive integer.
+
+     If [bound] is less than 2{^30}, [Random.int63 bound] is equal to
+     {!Random.int}[ bound]. If [bound] is greater than 2{^30} (on 64-bit systems
+     or non-standard environments, such as JavaScript), [Random.int63] returns
+     a value, where {!Random.int} raises {!Invalid_argument}.
```

I started with this, wrote the JavaScript comment, and then realised that there is a problem on JavaScript for bounds > 0x3FFFFFFF and <= 0x7FFFFFFF